tools/kvmexit.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/env python
+#
+# kvmexit.py
+#
+# Display the exit_reason and its statistics of each kvm_exit
+# for all vcpus of all virtual machines. For example,
+# $./kvmexit.py
+#  TGID     PID      KVM_EXIT_REASON                     STAT
```

Comment:
Instead of STAT, let us use COUNT, which states what the data is more accurately.

Reply:
> Instead of STAT, let us use COUNT, which states what the data is more accurately.

No problem, will reword this and the following.